infra/resources/modules/commons/servicebus.tf

```diff
@@ -22,6 +22,5 @@ resource "azurerm_servicebus_topic" "events" {
   name         = "${local.domain}-topic-events"
   namespace_id = azurerm_servicebus_namespace.main.id
 
-  enable_partitioning = true
-  support_ordering    = true
```

Comment:
I've found [this](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/servicebus_topic.html#enable_partitioning):
> Partitioning is available at entity creation for all queues and topics in Basic or Standard SKUs. It is not available for the Premium messaging SKU, but any previously existing partitioned entities in Premium namespaces continue to work as expected. Please [see the documentation](https://docs.microsoft.com/azure/service-bus-messaging/service-bus-partitioning) for more information.

Reply:
Exactly 👍 , this is why we are going to remove the `enable_partitioning` attribute